cohortextractor/tpp_backend.py

```diff
@@ -1047,10 +1059,10 @@ def _number_of_episodes_by_clinical_event(
             codelist, case_sensitive=True
         )
         date_condition, date_joins = self.get_date_condition(
-            "CodedEvent", "ConsultationDate", between
+            from_table, "ConsultationDate", between
         )
         ignored_day_condition, extra_queries = self._these_codes_occur_on_same_day(
-            "CodedEvent", ignore_days_where_these_codes_occur, between
+            from_table, ignore_days_where_these_codes_occur, between
         )
```

Comment:
Github won't let me comment on it directly because it's not part of the diff, but the `_these_codes_occur_on_same_day()` function has references that need fixing too.